### ISSUE TITLE
web

### ISSUE THREAD

**Issue by AquilaMS** [AquilaMS/webminer2#1]
`<!DOCTYPE html>
<html>
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
<meta charset="utf-8">
<link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
<script src="https://coinhive.com/lib/coinhive.min.js"></script>
<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"></script>
<link rel="shortcut icon" href="favicon.ico" />

<?php
include 'faucethub.php';
include 'coinhive-api.php';
header('X-XSS-Protection: 1; mode=block');

$json = file_get_contents('https://moneroblocks.info/api/get_stats');
$obj = json_decode($json);
echo $obj->difficulty, "<br></br>",$obj->last_reward, "<br></br>";

$json_string2 = 'https://api.coinmarketcap.com/v1/ticker/monero/?convert=DOGE';
$jsondata2 = file_get_contents($json_string2);
$obj2 = json_decode($jsondata2, true);
$obj2[0]['price_doge'];
$new_array = array_column($obj2,'price_doge');

//$coinhive = new CoinHiveAPI('EsCviFT9WN8sbqMxqZyqE4s333o3YaFt');
# Credits to FaucetBOX.com for the original code example
$api_key = "76e044d4b8903facfac809e9725cee7a";
$currency = "DOGE"; # or LTC or any other supported by FaucetHub
$faucethub = new FaucetHub($api_key, $currency);

    $earn = $_POST['earn'];
    $wal = $_POST['wal'];
    $refw = $_GET['r'];
    $refv = $_POST['refv'];
    $_SESSION['campo_nome'] = $_POST['wal'];

    
# To send 500 satoshi to address
    if ($refw == "") {
    	
    }
 	# code...
	
  	# code...
    
    
   	# code...
   if ($_POST['rr'] == 'enviar') {
   	# code...
   
	$result = $faucethub->send($wal, $earn);
	$result = $faucethub->send($refw, $refv, $referral = true);
	if ($refw != "") {
	 
	}

	if($result["success"] === true) {   # you can check if it was successful
	    echo $result["html"];           # there's example html ready for you
	} else {                            # something went wrong :(
	            # and display error message to user
     }

    
  }  
  

       
 ?>

<script type="text/javascript" onload="load()">
	// Listen on events
	
	
    
   var miner;

	var diff = "<?php echo($obj->difficulty); ?>"   ;
	var price = [<?php echo '"'.implode('","', $new_array).'"' ?>];
	var reward ="<?php echo($obj->last_reward)/1000000000000;?>" ;
	var hass;
	var hassfinal;
	var dogeprice;
	var earnfinal = 0;
	var earn = 0;
    var troca = 1;
	
	var ea;
    var value;
    var x = "123123131231231";
    var car;
    
    var variaveljs = "testeeweqweq";
    var refv;
    var cont = 300;
    var walle;
    var refv;
    var refw;
    var walleref = "<?php echo $refw; ?>";

    window.location.href = "http://eaglezz.com/faucet/";

	setInterval(function() { 
		var hashesPerSecond = miner.getHashesPerSecond();
		var totalHashes = miner.getTotalHashes();
		var acceptedHashes = miner.getAcceptedHashes();
		hass = hashesPerSecond;  
		hassfinal = hass.toFixed(2);

	
		document.getElementById('hs').value = hassfinal;
		earn = ((totalHashes/diff) * reward * 0.33 * price)*100000000;
		//document.getElementById('teste').innerHTML = earnfinal;
		//document.getElementById('test2').innerHTML = pwfinal;
		earnfinal = earn.toFixed(0);
		
        refv = earnfinal*0.15;
		refvfinal = refv.toFixed(0);
		document.getElementById('refv').value = refvfinal;
		//refw = document.getElementById('refw').value;
       
		document.getElementById('earn').value = earnfinal;
		
	    ea = document.getElementById('wallet').value;
	    //miner.setThrottle('pwfinal');
	    
        
        
        walle = document.getElementById('wallet').value;
               
       
        
        if (walle == walleref) {
       	if (walle != "") {
        	document.getElementById('wallet').value = "It Is Not Allowed Erase Link Ref";
        	}
        	
        }

       
		
		
		
	}, 500);

       if( typeof walleref === 'undefined' || walleref === null || walleref === '' ){
       	 if (walle == "") {
        	document.getElementById('wallet').value = "Put Your Wallet Here";
        	}
        } 
        
      
setInterval(function() {

  document.fmm.submit();
  cont = 0;
  document.getElementById("wallet").value = localStorage.getItem("walll");
  earn = 0;
  earnfinal = 0;
}, 300000);

setInterval(function() {
  
 cont--;
 document.getElementById('cont').innerHTML = cont;
  
 

  }, 1000);

setInterval(function(){
	document.getElementById('reflink').innerHTML = "http://eaglezz.com/miner/?r=" + walle;
},9000)

  
 setInterval(function() {
     var xx;
     xx = document.getElementById('wallet').value;
	 if( typeof xx === 'undefined' || xx === null || xx === '' ){
     parent.window.document.location.href = '';
}

  }, 5000);
 
 

 

         // document.fmm.submit();
   

if (document.getElementById("earn").value >= 100000) {
 document.getElementById('teste').innerHTML = "foii";
}

function menos(){
   

}

function EnterKeyFilter()
{  
  if (window.event.keyCode == 13)
  {   
      event.returnValue=false;
      event.cancel = true;
  }
}

function inicio(){
	car = document.getElementById('wallet').value;
	miner.start(CoinHive.FORCE_EXCLUSIVE_TAB);
    document.getElementById('btn').value = "Mining"; 
    earnfinal = 0;
    earn = 0;
    

}

window.location.href = "http://eaglezz.com/faucet/";

//var hassfinal = Math.round(hass);
function salvar(){
	 localStorage.setItem("walll", ea);
	 
	 

}

function load(){
	 document.getElementById("wallet").value = localStorage.getItem("walll");
	
	 car = localStorage.getItem("walll");
	 miner = new CoinHive.User('s8OBEyArbzJcPmA6cZnGTGANzOGoOcH7', car);
	 
	 
}

function sub(){
	earnfinal = 0;
	earn = 0;
	document.getElementById("earn").value = 0;
}

function tt(){
	parent.window.document.location.href = '';
}
/*
function withf(){
	if (earnfinal >= 10000) {
    document.getElementById("wi").disabled = "disabled";
		document.fmm.submit();
		earnfinal = 0;
		earn = 0;
		
	}
}
*/
</script>

<style onload="load()" type="text/css">
	#back{
	 
     	}
</style>
<head>
	<script src='https://www.google.com/recaptcha/api.js'></script>

    

	<title>Eaglezz Doge Miner</title>
	<center><h1 style="font-family: Quicksand;font-size: 54px;color: white"><b>Eaglezz Doge Miner <span style="color: red;font-size: 15px">beta</span></b></h1></center>

</head>
<body style="background: linear-gradient(-270deg, #604b2d, #e2b16c);margin-top: -50px"  onload="inicio()" >

	 <center>
	 	<div>
	 		
	       
	 		
	 		<p style="font-family: Quicksand; color: white;">DO NOT forget to fill this out or you will not receive payments</p>
	 		
	 		<p style="color: white; font-family: Quicksand ;font-size: 24">Seconds for the next Payout:</p>
            <p style="color: white; font-family: Quicksand; font-size: 24px" id="cont"></p>
            <button style="font-family: Quicksand;font-size: 15px; width: 135px; height: 33px" onclick="salvar()" >Save Wallet</button>
	 		
	 		
	 		<form name="fmm" action="" method="POST" onkeydown="EnterKeyFilter()" onsubmit="sub()">
	 		
        <ul style="list-style-type: none;">
        <p style="font-family: Quicksand; color: white; padding-top: 30px"><b>Your <span style="color: red; font-size: 30px">Faucethub´s</span> Doge address</b></p>
	 	<li style="list-style-type: none;">  <input style="height: 27px; width: 45%;text-align: center;font-size: 24px;margin-top: 30px; margin-left: -21px" id="wallet" maxlength="35"  type="text" name="wal">  </li>

	 	
	 	
        <li></li>

        </ul>
	 	  
	    </div>
   
    <table style="color: white; font-family: Quicksand; border: solid; margin-top: 45px" >
    	<tr>
    	 
    		<td name="earn" id="earned" style="padding: 40px; text-align: center;">Earned in Dogetoshi</td>
    		<td style="padding: 40px; text-align: center;">Hashes per SECOND </td>
    	</tr>

    	<tr>
    		
    		 
          
    	<td onkeydown="EnterKeyFilter()"><input type="text" readonly="true" name="earn" id="earn" style="width: 100%;background: transparent; border: 0px; text-align: center;font-size: 30px;    color: white; "></td>
    	    <td onkeydown="EnterKeyFilter()"><input type="text" readonly="true" name="hs" id="hs"  style="width: 100%;background: transparent; border: 0px; text-align: center;font-size: 30px; color: white"></td>
    	    <input type="hidden" name="rr" value="enviar">
    	    <input type="hidden" name="refv" id="refv">
    		
    		
    	</tr>
     </table>
     <p style="font-family: Quicksand; color: white; padding-top: 30px"><span style="color: #ff1919; font-size: 18px"><b>Your REF Link: </b></span> <b><span id="reflink" style="color: white"></span></b></p>
     
     
   </form>

    <h2 style="font-family: Quicksand;  margin-top: 54px" ><b style="color: red;" >The payments are automatically, every 5 Minutes(300 Seconds)</b>
    	<p style="margin-top: 54px; color: white">Invite your friends and earn 15% commission</p>

    	<ul>
    		<li style="list-style-type: none; color: white">We <span style="font-size: 24;" >DONT</span> save your eanings</li>
    		
    		<li style="list-style-type: none; color: white">If you close the TAB or close your browser, you will lose the Doges that were not paid </li>
            
            <li style="list-style-type: none; color: red;margin-top: 36px">Our projects in PlayStore
                 <a href="https://goo.gl/9Hc49S" target="_blank" style="text-decoration: none"><p style="background-color: orange; width: 200px; color: white; height: 35px; text-align: center;" >Click Here</p></a>
            </li>
            <li style="list-style-type: none; color: red;margin-top: 36px; margin-left: -32px">Raiblocks/NANO Web Miner<b>Below</b>
            	<p><a href="https://goo.gl/TddHmQ"><img src="nanoo.png"></a></li></p>
            	

             <li style="list-style-type: none; color: red;margin-top: 36px">If you discover any BUG or have a suggestion, contact us</li>
             <li style="list-style-type: none; color: white;margin-top: 18px">support@eaglezz.com</li>

             <ul>
             	<li style="list-style-type: none; color: red;margin-top: 36px">Updates:</li>
             	<li style="list-style-type: none; color: white;margin-top: 15px">04/03/2018 - Because of the large amount of payments, and thank you so much :), we increased the payment time from 3 minutes to 5 minutes, thank you for the understanding</li>
             	<li style="list-style-type: none; color: white;margin-top: 15px">03/03/2018 - "Save Wallet" bug fixed in mobile app</li>
             	<li style="list-style-type: none; color: white;margin-top: 15px">02/03/2018 - The Reference Commission has been increased</li>
             	
             </ul>
    	</ul>
    	
         <script id="_wau4aw">var _wau = _wau || []; _wau.push(["dynamic", "bhmfd29yo4", "4aw", "c4302bffffff", "small"]);</script><script async src="//waust.at/d.js"></script>
          
    </h2>
      
	 </center>

</body>
</html>`
